src/mlj_flux.jl

```diff
@@ -476,28 +384,35 @@ function MLJFlux.train(
 )
     X = X isa Tables.MatrixTable ? MLJBase.matrix(X) : X
 
-    la = LaplaceRedux.Laplace(
-        chain;
-        likelihood=:classification,
-        subset_of_weights=model.subset_of_weights,
-        subnetwork_indices=model.subnetwork_indices,
-        hessian_structure=model.hessian_structure,
-        backend=model.backend,
-        σ=model.σ,
-        μ₀=model.μ₀,
-        P₀=model.P₀,
-    )
-    verbose_laplace = false
+    if !isa(chain, AbstractLaplace)
```

Comment:
@pat-alt  why this check? when does chain  turns in a laplace object to require this check?

Reply:
Hey @Rockdeldiablo, please see my comment below, that should clarify this.